client/my-sites/stats/post-performance/index.jsx

```diff
@@ -112,12 +113,31 @@ const StatsPostPerformance = React.createClass( {
 										} )
 									}
 								</p>
-							) : null
-						}
-					</div>
-					<StatsTabs>
-						{ this.buildTabs( summaryUrl ) }
-					</StatsTabs>
+							</div>
+						) : null
+					}
+					{ ! post && ! isRequesting
+						? (
+							<div className="module-content-text is-empty-message is-error">
+								<p className="is-empty-message">
+									{ this.translate( 'You haven\'t published any posts yet.' ) }
+								</p>
+								<p>
+									<a href={ newPostUrl } className="button is-primary">
```

Comment:
Actually, It's is just a link. There's nothing done here apart going to the "new post" page.